gef.py

```diff
@@ -5237,186 +5421,43 @@ class PCustomCommand(GenericCommand):
 
     def __init__(self) -> None:
         super().__init__(prefix=True)
-        self["struct_path"] = (os.sep.join( (gef.config["gef.tempdir"], "structs")), "Path to store/load the structure ctypes files")
+        self["struct_path"] = (str(pathlib.Path(gef.config["gef.tempdir"]) / "structs"), "Path to store/load the structure ctypes files")
         self["max_depth"] = (4, "Maximum level of recursion supported")
         self["structure_name"] = ("bold blue", "Color of the structure name")
         self["structure_type"] = ("bold red", "Color of the attribute type")
         self["structure_size"] = ("green", "Color of the attribute size")
         return
 
-
-    def do_invoke(self, argv: List[str]) -> None:
-        argc = len(argv)
-        if argc == 0:
+    @parse_arguments({"type": "", "address": ""}, {})
+    def do_invoke(self, *args: Any, **kwargs: Dict[str, Any]) -> None:
```

Comment:
we don't use `*args`, so how about

```suggestion
    def do_invoke(self, _: Any, **kwargs: Dict[str, Any]) -> None:
```

Reply:
wouldn't it have to be `*_`?